PowerEditor/src/Notepad_plus.cpp

```diff
@@ -501,21 +501,55 @@ LRESULT Notepad_plus::init(HWND hwnd)
 	// Add external languages to menu
 	for (int i = 0; i < nppParam.getNbExternalLang(); ++i)
 	{
+		HMENU subMenu = hLangMenu;
 		ExternalLangContainer & externalLangContainer = nppParam.getELCFromIndex(i);
 
-		int numLangs = ::GetMenuItemCount(hLangMenu);
-		TCHAR buffer[menuItemStrLenMax]{};
+		int nbItem = ::GetMenuItemCount(subMenu);
+		TCHAR buffer[MAX_EXTERNAL_LEXER_NAME_LEN]{TEXT('\0')};

```

Comment:
Rather `TCHAR buffer[MAX_EXTERNAL_LEXER_NAME_LEN]{};` for conforming to the project coding style/rule.